ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1079,3 +1080,31 @@ literals:
 - zone_air_no2_probability_percentage_sensor
 - zone_air_no2_ozone_concentration_sensor
 
+# Added on 2021/10/22 for lighting control
+- lighting_switch_command:
+  - ON
+  - OFF
+- zone_lighting_switch_status:
+  - ON
+  - OFF
+- zone_lighting_switch_command:
+  - ON
+  - OFF
+- zone_brightness_percentage_command
+- lighting_switch_alarm:
+  - ACTIVE
+  - INACTIVE
+
+# Added on 2021/11/25 for water leak
+- water_leak_cable_distance_sensor
+
+#added for new Pre-action Fire Supression system
+- supervisor_alarm:
+  - ACTIVE
+  - INACTIVE
+- water_flowrate_status:
+  - ACTIVE
+  - INACTIVE
+- air_pressure_status:
```

Comment:
I did not see this used anywhere else in the PR. Is this needed?

Reply:
hi sir. it is used in SAFETY/entity_types/FACP.yaml for (PAFAS:)

Reply:
ah i see it. thank you for clarifying.